src/Common/tests/System.Net/Sockets/Performance/SocketPerformanceTests.cs

```diff
@@ -8,7 +8,7 @@
 using Xunit;
 using Xunit.Abstractions;
 
-namespace System.Net.Sockets.Performance.Tests
+namespace System.Net.Sockets.Tests
```

Comment:
Our naming convention is very clear: Unit.Tests, Functional.Tests and Performance.Tests. 
Renaming them would make their purpose unclear. Please revert these changes.